client/components/payment-confirm-illustration/index.tsx

```diff
@@ -10,7 +11,14 @@ import classNames from 'classnames';
  */
 import './styles.scss';
 
-const PaymentConfirmIllustration = ( { hasBorder, Icon = () => null } ) => {
+const PaymentConfirmIllustration = ( {
+	hasBorder,
+	Icon = () => <React.Fragment />,
```

Comment:
can we rename it to `icon` to comply with the naming convention?

Reply:
I don't think it's possible given it's used as a JSX property afterward `<Icon />`, so it should start with a capital letter.

One thing we can do is make it lowercase, but once we want to render it, assign it to another variable with a capital letter. Something like `const Icon = icon;`.

It looks like a weird fix, but I agree that `props` should be lowercase.

Reply:
Okay, can you create a discussion thread in the team's private channel so we could align on if we should touch this, please?